code/modules/client/preference_setup/loadout/loadout_general.dm

```diff
@@ -120,6 +120,52 @@
 	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(lunchboxes))
 	gear_tweaks += new/datum/gear_tweak/contents(lunchables_lunches(), lunchables_snacks(), lunchables_drinks())
 
+/datum/gear/coffeemug
+	display_name = "coffee mugs"
+	description = "A coffee mug in various designs."
+	cost = 1
+	path = /obj/item/weapon/reagent_containers/food/drinks/glass2/coffeemug
+
+/datum/gear/coffeemug/New()
```

Comment:
same as on `/obj/random/mug/item_to_spawn()`

Reply:
Not sure about using it here since it would include tall mugs and the glass mug which imo are a pretty majorly different vibe from normal run of the mill ceramic mugs.

Reply:
Fair; is only for thought